contrib/verifybinaries/verify.py

```diff
@@ -63,6 +71,17 @@ def download_with_wget(remote_file, local_file=None):
     return result.returncode == 0, result.stdout.decode().rstrip()
 
 
+def download_lines_with_urllib(url) -> (bool, [str]):
+    """Get (success, text lines of a file) over HTTP."""
+    try:
+        return True, [l.strip().decode() for l in urllib.request.urlopen(url).readlines()]
+    except urllib.request.HTTPError:
+        print_warn(f"HTTP request to {url} failed (HTTPError)")
+    except Exception as e:
+        print_warn(f"HTTP request to {url} failed ({e})")
+    return (False, [])
```

Comment:
Pico-nit in case you re-touch: fn returns an unparenthesized tuple on L77 and parenthesized here. There's no behavioural difference in Python, but could be nice to have them the same (as "the comma makes the tuple", perhaps both unparenthesized?)